src/components/layout.css

```diff
@@ -392,9 +392,26 @@ main {
   font-weight: 400;
   line-height: 1.58;
   letter-spacing: -.003em;
-  max-width: 768px;
   padding-left: 4.2rem;
-  width: calc(100% - 36.0rem);
+}
+
+.article-reader__inner{
+    max-width: 768px;
+    width: calc(100% - 36.0rem);
```

Comment:
Nit: these CSS here seem to be over-indented? (don't see the toolset has configs to lint CSS yet, so perhaps CSS format is trivial)

Reply:
Ah. True. I will keep this in mind.